### ISSUE TITLE
[Idea] Shoot down goon barbs

### ISSUE THREAD

**Issue by slipher** [Unvanquished/gameplay#31]
It should be possible to shoot goon barbs and make them blow up in midair.

Why: because it would be cool. I feel like the game is a little lacking in interactions between different elements. A lot of stuff just goes through each other.

Open questions: Should the barb do splash damage or not? If so, should the damage be attributed to the one who launched the barb or the one who blew it up in midair?

**Comment by Viech** [comment]
Preventing a snipe against a buildable, assuming that we even want that given the alien's lackluster anti-base options, would be insanely difficult unless you are standing in front or on top of that buildable (i.e. when you are a dedicated defender who is already causing enough frustration to the attackers). In combat oth humans could randomly hit a projectile that is fired directly at them, which is already a very difficult trick for the Goon to pull off and should rather be rewarded when the aim is on point. So it seems to me that this change would turn the nobs in the wrong direction.

_The same idea with respect to rockets I could see, though unfortunately alien's can't shoot._

**Comment by slipher** [comment]
> Preventing a snipe against a buildable, assuming that we even want that given the alien's lackluster anti-base options, would be insanely difficult

Yes, that's what I expect. I agree that we wouldn't want it to be easy to stop barbs against the base. I want it more for the cool factor rather than to have a big effect on gameplay.

> In combat oth humans could randomly hit a projectile that is fired directly at them, which is already a very difficult trick for the Goon to pull off and should rather be rewarded when the aim is on point

If we opt to have the barb do the usual amount of splash damage, then the human could still take a good amount of damage if he blows it up right in front of his face.

**Comment by Gireen** [comment]
At the first glance this idea sounds fun.
But the problems it creates have to much negative effects. 

It would give humans more incentive to camp, trying  to shoot down barbs.
They are so small and fast that shooting them down is more a matter of luck, but highly skilled players or aimbots would make barbs useless.
Would anyone even notice that they shot down a barb?

**Comment by DolceTriade** [comment]
I think it's a cool idea. Shooting barbs down seems hard.
